Fix flake8 issues

Flake8 is currently running in non-fatal mode in our CI because of some issues.
We should fix the issues and make flake8 fail our pipeline again.

```
$ flake8 --exclude=build,venv --ignore= --max-line-length=200 --max-complexity=75 --show-source --statistics .
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/2.7.18/x64
./lib/cfv/osutil.py:112:78: E741 ambiguous variable name 'I'
def strippath(filename, num='a', _splitdrivere=re.compile(r'[a-z]:[/\\]', re.I)):
                                                                             ^
./lib/cfv/common.py:738:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/common.py:900:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/common.py:1400:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/common.py:1505:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/common.py:1537:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/common.py:1572:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/common.py:1649:34: E741 ambiguous variable name 'l'
    def do_test_chksumline(self, l):
                                 ^
./lib/cfv/strutil.py:6:14: E741 ambiguous variable name 'l'
def safesort(l):
             ^
./test/cfvtest.py:49:26: E741 ambiguous variable name 'l'
    def writelines(self, l):
                         ^
10    E741 ambiguous variable name 'l'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -57,7 +57,7 @@ jobs:
         run: check-manifest .
       
       - name: Run flake
-        continue-on-error: true
+        continue-on-error: false
         run: flake8 --exclude=build,venv --ignore= --max-line-length=200 --max-complexity=75 --show-source --statistics .
           
       - name: Check distribution

--- a/lib/cfv/common.py
+++ b/lib/cfv/common.py
@@ -735,10 +735,10 @@ class FooSum_Base(TextChksumType):
     def do_test_chksumfile_print_testingline(self, file):
         TextChksumType.do_test_chksumfile_print_testingline(self, file, parse_commentline(file.peekline(512), ';#'))
 
-    def do_test_chksumline(self, l):
-        if l[0] in ';#':
+    def do_test_chksumline(self, line):
+        if line[0] in ';#':
             return
-        x = self._foosum_rem.match(l)
+        x = self._foosum_rem.match(line)
         if not x:
             return -1
         if x.group(2) == ' ':
@@ -897,8 +897,8 @@ class BSDMD5(TextChksumType, MD5_MixIn):
 
     _bsdmd5rem = re.compile(r'MD5 \((.+)\) = ([0-9a-fA-F]{32})[\r\n]*$')
 
-    def do_test_chksumline(self, l):
-        x = self._bsdmd5rem.match(l)
+    def do_test_chksumline(self, line):
+        x = self._bsdmd5rem.match(line)
         if not x:
             return -1
         self.test_file(x.group(1), unhexlify(x.group(2)))
@@ -1397,10 +1397,10 @@ class SFV_Base(TextChksumType):
         # override the default testing line to show first SFV comment line, if any
         TextChksumType.do_test_chksumfile_print_testingline(self, file, parse_commentline(file.peekline(512), ';'))
 
-    def do_test_chksumline(self, l):
-        if l[0] == ';':
+    def do_test_chksumline(self, line):
+        if line[0] == ';':
             return
-        x = self._sfvrem.match(l)
+        x = self._sfvrem.match(line)
         if not x:
             return -1
         self.test_file(x.group(1), unhexlify(x.group(2)))
@@ -1502,8 +1502,8 @@ class CSV(TextChksumType, CRC_MixIn):
 
     _csvrem = re.compile(_csvfnre + r'([0-9]+),([0-9a-fA-F]{8}),')
 
-    def do_test_chksumline(self, l):
-        x = self._csvrem.match(l)
+    def do_test_chksumline(self, line):
+        x = self._csvrem.match(line)
         if not x:
             return -1
         self.test_file(csvunquote(x.group(1), x.group(2)), unhexlify(x.group(4)), int(x.group(3)))
@@ -1534,8 +1534,8 @@ class CSV4(TextChksumType, CRC_MixIn):
 
     _csv4rem = re.compile(r'%s([0-9]+),([0-9a-fA-F]{8}),%s' % (_csvfnre, _csvstrre))
 
-    def do_test_chksumline(self, l):
-        x = self._csv4rem.match(l)
+    def do_test_chksumline(self, line):
+        x = self._csv4rem.match(line)
         if not x:
             return -1
         name = csvunquote(x.group(1), x.group(2))
@@ -1569,8 +1569,8 @@ class CSV2(TextChksumType):
 
     _csv2rem = re.compile(_csvfnre + r'([0-9]+),')
 
-    def do_test_chksumline(self, l):
-        x = self._csv2rem.match(l)
+    def do_test_chksumline(self, line):
+        x = self._csv2rem.match(line)
         if not x:
             return -1
         self.test_file(csvunquote(x.group(1), x.group(2)), None, int(x.group(3)))
@@ -1646,11 +1646,11 @@ class JPEGSheriff_CRC(TextChksumType, CRC_MixIn):
     _commentboundary = re.compile(r'^-+(\s+-+){1,4}\s*$')
     _nstrip = re.compile(r'[.,]')
 
-    def do_test_chksumline(self, l):
-        if self._commentboundary.match(l):
+    def do_test_chksumline(self, line):
+        if self._commentboundary.match(line):
             self.in_comments = not self.in_comments
             return
-        x = self._crcrem.match(l)
+        x = self._crcrem.match(line)
         if not x:
             if self.in_comments:
                 return

--- a/lib/cfv/osutil.py
+++ b/lib/cfv/osutil.py
@@ -109,7 +109,7 @@ def path_split(filename):
     return parts
 
 
-def strippath(filename, num='a', _splitdrivere=re.compile(r'[a-z]:[/\\]', re.I)):
+def strippath(filename, num='a', _splitdrivere=re.compile(r'[a-z]:[/\\]', re.IGNORECASE)):
     """Strip off path components from the left side of the filename.
 
     >>> strippath(os.path.join('c:','foo','bar','baz'))

--- a/lib/cfv/strutil.py
+++ b/lib/cfv/strutil.py
@@ -3,17 +3,17 @@ import unicodedata
 from StringIO import StringIO
 
 
-def safesort(l):
+def safesort(line):
     sl = []
     ul = []
-    for s in l:
+    for s in line:
         if isinstance(s, str):
             sl.append(s)
         else:
             ul.append(s)
     sl.sort()
     ul.sort()
-    l[:] = ul + sl
+    line[:] = ul + sl
 
 
 def showfn(s):

--- a/test/cfvtest.py
+++ b/test/cfvtest.py
@@ -46,7 +46,7 @@ class NullFile:
     def write(self, s):
         pass
 
-    def writelines(self, l):
+    def writelines(self, lines):
         pass
 
     def flush(self):

